src/families/tron/Votes/index.js

```diff
@@ -0,0 +1,244 @@
+// @flow
+import React, { useEffect, useState, useCallback } from "react";
+import { View, Linking, TouchableOpacity, StyleSheet } from "react-native";
+import { Trans } from "react-i18next";
+import type { Account } from "@ledgerhq/live-common/lib/types";
+
+import { formatCurrencyUnit } from "@ledgerhq/live-common/lib/currencies";
+import {
+  getTronSuperRepresentatives,
+  getNextVotingDate,
+} from "@ledgerhq/live-common/lib/api/Tron";
+
+import { BigNumber } from "bignumber.js";
+import moment from "moment";
+import { urls } from "../../../config/urls";
+
+import Row from "./Row";
+import Header from "./Header";
+import LText from "../../../components/LText";
+import Button from "../../../components/Button";
+import colors from "../../../colors";
+import ExternalLink from "../../../icons/ExternalLink";
+import Info from "../../../icons/Info";
+
+type Props = {
+  account: Account,
+  parentAccount: ?Account,
+};
+
+// @TODO move this to common
```

Comment:
can we have a PR that do this in live-common? unlike what we did in tezos, I suggest to have a tron/react.js in the live-common family to not put the react code in the logic.